jme3-desktop/src/main/java/com/jme3/cursors/plugins/CursorLoader.java

```diff
@@ -107,7 +109,7 @@ private JmeCursor loadCursor(InputStream inStream) throws IOException {
             int steps = 0;
             int width = 0;
             int height = 0;
-            int flag = 0; // we don't use that.
+            //int flag = 0; // we don't use that.
```

Comment:
Why comment out rather than delete?

Reply:
Good Question. I have no strong opinion on keeping the comment :)

Reply:
Since everything's under source control, code should be commented out only if there's a plausible reason to un-comment it later. And that rationale only works if it's obvious to someone reading the code what effects un-commenting would have.